specs/delete_spec.js

```diff
@@ -0,0 +1,27 @@
+let MenuPage = require ("../page_objects/menu_page")
```

Comment:
user lowercase as it is variable `menuPage `

Reply:
fixed